crates/cairo-lang-runner/src/lib.rs

```diff
@@ -522,6 +523,35 @@ impl SierraCasmRunner {
     }
 }
 
+pub fn print_result(value: RunResultValue){
```

Comment:
```suggestion
pub fn print_result(value: &RunResultValue){
```

you don't have to take ownership of value here